server/src/main.cc

```diff
@@ -1,45 +1,54 @@
-#include <memory>
-
 #include <oatpp/core/base/Environment.hpp>
-#include <oatpp/core/macro/component.hpp>
-#include <oatpp/core/provider/Provider.hpp>
-#include <oatpp/network/ConnectionHandler.hpp>
-#include <oatpp/network/ConnectionProvider.hpp>
-#include <oatpp/network/Server.hpp>
-#include <oatpp/web/server/HttpRouter.hpp>
 #include <oatpp/web/server/api/Endpoint.hpp>
+#include <oatpp/network/Server.hpp>
+#include <oatpp-sqlite/Connection.hpp>
+#include <oatpp/core/macro/component.hpp>
 #include <oatpp-swagger/Controller.hpp>
 
-#include "component/app_component.hh"
-#include "controller/user_controller.hh"
-#include "controller/offer_controller.hh"
+#include "component/AppComponent.hh"
+#include "controller/AuthController.hh"
+#include "controller/OfferController.hh"
+#include "controller/UserController.hh"
+#include "controller/PlaceController.hh"
 
 int
-main()
-{
-    ::oatpp::base::Environment::init();
+main() {
+    oatpp::base::Environment::init();
 
-    ::server::component::app_component const app_component{};
+    server::component::AppComponent appComponent;
 
-    OATPP_COMPONENT(::std::shared_ptr<::oatpp::web::server::HttpRouter>, router);
+    auto router = appComponent.httpRouterComponent.getObject();
```

Comment:
I think it wasn't my change but @ArtMan0's. I don't know if it's a good practice to access a component by other component instead of calling the macro.

Reply:
I changed it back